src/App.tsx

```diff
@@ -5,7 +5,18 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const onAddMovie = (newMovie: Movie) => {
```

Comment:
In the upper component - handle, in the down - on
```suggestion
  const handleMovie = (newMovie: Movie) => {
```